BaseTest.java

```diff
@@ -0,0 +1,28 @@
+package common;
+
+import lombok.Getter;
+import org.junit.After;
+import org.junit.Before;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+
+@Getter
+public abstract class BaseTest {
+
+    protected WebDriver webDriver;
+    protected PropertyHelper propertyHelper = new PropertyHelper();
+
+    @Before
+    public void initDriver() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+    }
+
+    @After
+    public void shutDownDriver() {
+        webDriver.close();
```

Comment:
what is the diff between close & quite? why do you need both in this case?

Reply:
Yes, I know. Close() - used to close the current open window. Quit() - used to destroy the instance of webdriver. It is possible not to use both.